Cd week 1/c hello world/adebola

Ade0211 opened the pull request:
# Your Details

Your Name: Adebola Alaba_Ige
Your City: Wolverhampton
Your Slack Name: Adebola Alaba_Ige

# Homework Details

Module:
Week: 1

Ade0211 committed:
exercises

sorted out the step by step exercises.

Ade0211 committed:
update

updating my exercises

Ade0211 committed:
still updating

Ade0211 committed:
magic 8 ball

worked on my magic 8 ball

pads commented on week-1/1-exercises/I-floats/exercise.js:
```diff
@@ -1,2 +1,9 @@
 var numberOfMentors = 8;
+var totalNumber = 23;
+var roundedNumber1 = Math.round(numberOfStudents/totalNumber*100);
+var roundedNumber2 = Math.round(numberOfMentors/totalNumber*100);
```
Good job on this, it helps to add spacing around the operators (`/` and `*`) just to make it a bit easier to read.

pads commented on week-1/1-exercises/J-functions/exercise.js:
```diff
@@ -1,7 +1,20 @@
 function halve(number) {
   // complete the function here
-}
+return number /2;  
```
To help make the code easier to read, lines written inside functions should be indented.

Ade0211 replied:
i have worked on the feedback but it is showing that it is having conflict with the master repository

pads replied:
Don't worry about the conflicts as these changes are not intended to be merged.  Great job on fixing the remainder function! 👍

pads commented on week-1/2-mandatory/1-syntax-errors.js:
```diff
@@ -2,18 +2,24 @@
 
     // Use string interpolation here
-    return "The total is %{total}"
+    return `The remainder is 3`;
```
This passes the test but you shouldn't have 3 hard-coded here or the division by 6 above.  Can you figure out how to calculate a remainder in JavaScript? (you are free to look this up in a search engine by the way!)

Ade0211 replied:
Thanks so much for the feedback. I have worked on the feedback provided.

pads reviewed (approved):
Well done Adebola this is a great effort. I would encourage you to try and get the remainder function in the first mandatory exercise working without hard-coded numbers.  

I'd also make sure your code is indented consistently and add spacing around operators just to help with reading the code but these are minor points :)

Great to see you take on the extra exercises too!

Ade0211 committed:
Feedback worked on

I have just worked on the remainder function and also my indentation and spacing
